src/components/layout/navbar/navbar.css

```diff
@@ -0,0 +1,55 @@
+.navbar {
```

Comment:
general style guide:
- you did it well, this is just summary for you
  - keep the grid: https://trydesignlab.com/blog/grids-ui-ux-graphic-design-quick-history-5-amazing-tips/
    - if you have grid `10px` use multiplication of 10(x1, x2, x4, etc, and "halfs" x0.5, x1.5): 5px, 10px, 15px, 20px. 40px,
    - assume font sizes (eg. small 14, base: 16px, large: 18px, xl: 24px)
    - keep icons sizes (eg. small: 16px, base: 20px, large: 24px)
  - sort props alphabetically (if possible)